layouts/index.hbs

```diff
@@ -140,18 +148,18 @@
         }
       }
     </style>
-    
+
     <div class="interactive">
       <span id="interactive-first">
         <img src="/static/images/interactive/nodejs-interactive-logo-center.png" />
       </span>
-      <a href="http://events.linuxfoundation.org/events/node-interactive-europe">
+      <a id="interactive-left" href="http://events.linuxfoundation.org/events/node-interactive-europe">
```

Comment:
Can't we simply use `style="min-width:0;"` or add the same CSS class for both anchor elements?